sdk/anomalydetector/azure-ai-anomalydetector/src/samples/java/com/azure/ai/anomalydetector/ReadmeSamples.java

```diff
@@ -3,9 +3,43 @@
 
 package com.azure.ai.anomalydetector;
 
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.http.policy.AzureKeyCredentialPolicy;
+import com.azure.core.http.policy.HttpLogOptions;
+import com.azure.core.http.policy.HttpLogDetailLevel;
+import com.azure.core.http.ContentType;
+import com.azure.core.http.HttpClient;
+import com.azure.core.http.HttpHeaders;
+import com.azure.core.http.HttpPipeline;
+import com.azure.core.http.HttpPipelineBuilder;
+import com.azure.core.http.policy.AddHeadersPolicy;
+import com.azure.core.http.policy.HttpPipelinePolicy;
+
 public final class ReadmeSamples {
-    public void readmeSamples() {
-        // BEGIN: com.azure.ai.anomalydetector.readme
-        // END: com.azure.ai.anomalydetector.readme
+
+    private static void createClient() {
+        // BEGIN: readme-sample-createAnomalyDetectorClient
+        String endpoint = "<anomaly-detector-resource-endpoint>";
+        String key = "<anomaly-detector-resource-key>";
+
+        HttpHeaders headers = new HttpHeaders()
+            .put("Accept", ContentType.APPLICATION_JSON);
+
+        HttpPipelinePolicy authPolicy = new AzureKeyCredentialPolicy("Ocp-Apim-Subscription-Key",
+            new AzureKeyCredential(key));
+        AddHeadersPolicy addHeadersPolicy = new AddHeadersPolicy(headers);
+
+        HttpPipeline httpPipeline = new HttpPipelineBuilder().httpClient(HttpClient.createDefault())
+            .policies(authPolicy, addHeadersPolicy).build();
+        // Instantiate a client that will be used to call the service.
+        HttpLogOptions httpLogOptions = new HttpLogOptions();
+        httpLogOptions.setLogLevel(HttpLogDetailLevel.BODY_AND_HEADERS);
+
+        AnomalyDetectorClient anomalyDetectorClient = new AnomalyDetectorClientBuilder()
+            .pipeline(httpPipeline)
```

Comment:
any reason to choose use the HttpPipeline instead of using "credential()" method in builder?

Reply:
follow the previous version.

Reply:
This is the sample, so "follow the previous version." is not an answer. You need to know why.

Reply:
Hi @weidongxu-microsoft , why should we choose "credential" here?
Actually, we use ”AzureKeyCredential“ in HttpPipeline.

        _**HttpPipelinePolicy authPolicy = new AzureKeyCredentialPolicy("Ocp-Apim-Subscription-Key",
            new AzureKeyCredential(key));
        AddHeadersPolicy addHeadersPolicy = new AddHeadersPolicy(headers);
        HttpPipeline httpPipeline = new HttpPipelineBuilder().httpClient(HttpClient.createDefault())
            .policies(authPolicy, addHeadersPolicy).build();**_

Reply:
I had the same comment above in README. We should use `credential` here and not use HttpPipeline.

Reply:
@minkefusiji 

I think I even generated the samples for you about the "credential()"? https://github.com/Azure/azure-sdk-for-java/blob/cadl_dpg_anomalydetector/sdk/anomalydetector/azure-ai-anomalydetector/src/samples/java/com/azure/ai/anomalydetector/generated/GetMultivariateModel.java#L17-L21

Of course they may need changes, but at least take it as what you start?

Reply:
fixed it.